openmc/data/neutron.py

```diff
@@ -817,6 +830,43 @@ def from_njoy(cls, filename, temperatures=None, evaluation=None, **kwargs):
             ev = evaluation if evaluation is not None else Evaluation(filename)
             if (1, 458) in ev.section:
                 data.fission_energy = FissionEnergyRelease.from_endf(ev, data)
+                # Add 318 fission heating data from heatr
+                heatr = Evaluation(kwargs["heatr"])
```

Comment:
When multiple temperatures are requested, the ENDF files actually have multiple evaluations in then, one for each temperature. So, to get the kerma for each, you could do something like:
```Python
evals = openmc.data.endf.get_evaluations(kwargs["heatr"])
for ev in evals:
    file_obj = StringIO(ev.section[3, 318])
    get_heat_record(file_obj)
    ... = get_tab1_record(file_obj)
```

Reply:
That's where the temperatures are at! I saw that there were multiple ACE files made for each temperature, but couldn't nail this down for `heatr`, hence the really hacky way the multiple temperatures were accounted for currently. Will switch that over

Reply:
Yeah, it's not at all obvious. I had to scratch my head for a bit to remember how it all worked.

Reply:
Was able to pull the temperatures how you suggested [c7a8b66]. I am currently pulling the `temperature` from the `target` attribute on the evaluations, as I couldn't find it readily accessible. I thought `Reaction.from_endf` might work, but it is only configured for a single evaluation